### ISSUE TITLE
A11y_.NETOnlineREPL_Output_Screenreader: NVDA is not announcing the name of "Feedback" link in a descriptive way.

### ISSUE THREAD

**Issue by pranshu512** [dotnet/try#908]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**: #A11yMAS;#A11yTCS;#A11ySev2;#BM_.NET Online REPL_Web_Feb2021;#.NET Online REPL;#Win10;#FTP;#DesktopWeb;#WCAG2.4.4;#Screenreader;#Narrator;#NVDA;#Chrome;#ChromiumEdge;

### **Environment Details:**
Application Name: .NET Online REPL
Operating System: Windows10
Browser Version : Google Chrome Version 87.0.4280.141 (Official Build) (64-bit)
Chromium Edge Version 89.0.774.14 (Official build) dev (64-bit)
Screenreader: NVDA 2020.3,Narrator

### **Repro Steps:**

1. Open .NET Editor coding screen .
2. Start NVDA and tab to Feedback link .
3. Check whether NVDA announces the name of "Feedback" link in a descriptive way

### **Actual:**
NVDA is not announcing the name of "Feedback" link in a descriptive way while navigating using tab key.

Same issue is also observed with Chromium Edge +Narrator.

### **Expected:**
NVDA should announce the name of "Feedback" link in a descriptive way while navigating using tab key.
NVDA should also announce feedback for what purpose.

### **User Impact:**
Screen reader users will not be able to understand the correct purpose of the link.

**Comment by fsteffi** [comment]
@LadyNaggaga  please can you please close this bug as the name lets users know the purpose

**Comment by 12-shweta** [comment]
It is expected behavior and bug can be closed.

**Comment by fsteffi** [comment]
**githubtag**s: #Closed;
